spec/defines/collectd_plugin_curl_json_spec.rb

```diff
@@ -13,6 +13,9 @@
     {
       url: 'http://localhost:55672/api/overview',
       instance: 'rabbitmq_overview',
+      verifypeer: 'false',
+      verifyhost: 'false',
```

Comment:
False doesn't equal undef in manifest. If you don't set those variables, puppet won't set this parameter in collectd config and collectd will use the default, which is true for those two parameters. Should I change that to true?